scripts/arm32_ci_script.sh

```diff
@@ -0,0 +1,289 @@
+
+    exit 1
+}
+trap handle_ctrl_c INT
+
+#Trap Exit and handle it
+function handle_exit {
+    set +x
+
+    echo 'The script is exited. Cleaning environment..'
+
+    clean_env
+ }
+trap handle_exit EXIT
+
+#Mount with checking to be already existed
+function mount_with_checking {
+    set +x
+    local options="$1"
+    local from="$2"
+    local rootfsFolder="$3"
+
+    if mountpoint -q -- "$rootfsFolder"; then
+        (set +x; echo "$rootfsFolder is already mounted.")
+    else {
+        (set -x; sudo mount $options "$from" "$rootfsFolder")
+    }
+    fi
+}
+
+#Mount emulator to the target mount path
+function mount_emulator {
+    #Check if the mount path exists and create if neccessary
+    if [ ! -d "$__ARMRootfsMountPath" ]; then
+        sudo mkdir -p "$__ARMRootfsMountPath"
+    fi
+
+    set +x
+    mount_with_checking "" "$__ARMEmulPath/platform/$__ARMRootfsImageBase" "$__ARMRootfsMountPath"
+    mount_with_checking "-t proc" "/proc"    "$__ARMRootfsMountPath/proc"
+    mount_with_checking "-o bind" "/dev/"    "$__ARMRootfsMountPath/dev"
+    mount_with_checking "-o bind" "/dev/pts" "$__ARMRootfsMountPath/dev/pts"
+    mount_with_checking "-t tmpfs" "shm"     "$__ARMRootfsMountPath/run/shm"
+    mount_with_checking "-o bind" "/sys"     "$__ARMRootfsMountPath/sys"
+}
+
+# Cross builds core-setup using Docker image
+function cross_build_core_setup_with_docker {
+    # Cross building using Docker image
+    currentWorkingDirectory=`pwd`
+
+    # Build rootfs in docker
+    #dockerCmd="sudo docker run -i --rm -v $currentWorkingDirectory:/opt/core-setup -w /opt/core-setup chcosta/dotnetcore:ubuntu1404_cross_prereqs_v1"
+    #dockerCmd="sudo docker run -i --rm -v $currentWorkingDirectory:/opt/core-setup -w /opt/core-setup t2wish/dotnetcore:ubuntu1404_cross_prereqs_v2"
+    # Build rootfs in Docker
+    #if [ "$__buildArch" == "arm" ]; then
+    #    buildRootfsCmd="/opt/core-setup/cross/build-rootfs.sh arm"
+    #elif [ "$__buildArch" == "armel" ]; then
+    #    buildRootfsCmd="/opt/core-setup/cross/build-rootfs.sh armel tizen"
+    #else
+    #    echo "ERROR: unknown buildArch $__buildArch"
+    #    exit 1
+    #fi
+    #$dockerCmd $buildRootfsCmd
+
+    # Build rootfs in host
+    if [ "$__buildArch" == "arm" ]; then
+        sudo apt-get update
+        sudo apt-get -y install  qemu qemu-user-static binfmt-support debootstrap
+        buildRootfsCmd="sudo ./cross/build-rootfs.sh arm xenial --skipunmount"
+    elif [ "$__buildArch" == "armel" ]; then
+        sudo apt-get update
+        sudo apt-get -y install libxml2-utils rpm2cpio
+        buildRootfsCmd="sudo ./cross/build-rootfs.sh armel tizen --skipunmount"
+    else
+        echo "ERROR: unknown buildArch $__buildArch"
+        exit 1
+    fi
+    $buildRootfsCmd
+    sudo chown -R $(id -u -n) cross/rootfs
+
+    # Cross building core-setup with rootfs in Docker
+    if [ "$__buildArch" == "arm" ]; then
```

Comment:
I presume this PR needs to be updated in a similar manner as the CoreFX Pr for the CI script, right?